db/migrate/20160218161746_add_csv_export.rb

```diff
@@ -0,0 +1,12 @@
+class AddCsvExport < ActiveRecord::Migration
+  def change
+    create_table :csv_exports do |t|
+      t.integer :user_id,    null: false
+      t.timestamps           null: false
+      t.timestamp :deleted_at
+      t.string :content,        null: false
+      t.string :status
+      t.string :filename
```

Comment:
Was using this originally, but code climate was complaining about the filename coming from a model attribute with user input.  Switched it to use the id so its safer, and kept it to send a useful filename of the report.  It could have the report generated and completed time appended to the report, or use created_at in the filename.